rest-api/client_cli/rdr.py

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env python
+#
+# RDR cli tool launcher
+#
+
+# pylint: disable=superfluous-parens
+import copy
+import glob
+import importlib
+import os
+import sys
+
+lib_paths = ['../service_libs', 'service_libs']
+import_path = 'service_libs'
+
+
+def run():
+  args = copy.deepcopy(sys.argv)
+
+  show_usage = False
+  group = 'no-group'
+
+  # If help is select lets build a list of commands to show
+  if '--help' == sys.argv[1] or '-h' == sys.argv[1]:
+    show_usage = True
+
+  # If not showing help, get the group name and then we'll call it.
+  if not show_usage:
+    group = args.pop(1)
+    sys.argv = args
+
+  lp = None
+  for lib_path in lib_paths:
+    if os.path.exists(os.path.join(os.curdir, lib_path)):
+      lp = os.path.join(os.curdir, lib_path)
+
+  if not lp:
+    print('ERROR: service libs path not found, aborting.')
+    exit(1)
+
+  group_names = list()
+
+  libs = glob.glob(os.path.join(lp, '*.py'))
```

Comment:
What you say is true, but it doesn't add more than a second to the startup time.  I would rather see a file name like 'verify_environment.py' than just 'verify.py', and I would rather type 'rdr verify' on the command line instead of 'rdr verify_environment'.  If this is a sticking point for you, lets discuss more.

Reply:
This is not a sticking point for me. Just something to be aware of in the scripts we write. Imports and monkey patches from all scripts will execute when any script executes.

Reply:
I'll change this to a "grep the file" to get the "group" and "group_desc" vars.  That way its not importing anything until it is going to run something.

Reply:
I like that solution